ChangeLog

```diff
@@ -175,6 +175,10 @@ Release date: tba
 
       Closes #1087
 
+    * Do not warn about unused arguments in singledispatch registered implementations.
```

Comment:
Docs and PR should be renamed, since false-positive function-redefined was also removed. Maybe something generic like 'Improve support for singledispatch functions'?